src/initStarter.js

```diff
@@ -146,6 +194,74 @@ const writeReadme = async (rootPath, name, type) => {
   }
 };
 
+const replaceInFilesByType = async (patterns, template, newValue) => {
+  const options = {
+    files: patterns,
+    from: new RegExp(template, 'g'),
+    to: newValue,
+  };
+  try {
+    const res = await replaceInFile(options);
+    return res.filter((f) => f.hasChanged);
+  } catch (e) {
+    return [];
+  }
+};
+
+const replaceProjectName = async (rootPath, fullName, projectName) => {
+  const prevDir = process.cwd();
+  console.log('branding project...');
+  process.chdir(rootPath);
+  try {
+    // replace in yaml
+    const yamlFiles = await Promise.all(
+      ['.github/workflows/*.yaml', '.github/workflows/*.yml'].map(
+        async (pattern) =>
+          replaceInFilesByType(pattern, YAML_TEMPLATE, fullName)
+      )
+    );
+    console.log(
+      'Yaml files:',
+      yamlFiles.flat().map((f) => f.file)
+    );
+
+    // replace in html
+    const htmlFiles = await replaceInFilesByType(
+      'public/*.html',
+      HTML_TEMPLATE,
+      fullName
+    );
+    console.log(
+      'HTML files:',
+      htmlFiles.map((f) => f.file)
+    );
+
+    // replace name in package.json
+    replaceJSON('./package.json', 'name', projectName, { silent: true });
+    replaceJSON('./public/manifest.json', 'name', projectName, {
+      silent: true,
+    });
+    replaceJSON('./public/manifest.json', 'short_name', projectName, {
+      silent: true,
+    });
+  } catch (e) {
+    // todo: add logger
+    console.error(e);
+  } finally {
+    process.chdir(prevDir);
+  }
+};
+
+const checkDeploymentMethod = async (rootPath, useGithubActions) => {
+  if (!useGithubActions) {
+    try {
+      await fs.remove(path.join(rootPath, '.github'));
+    } catch (e) {
+      console.error(e);
+    }
+  }
+};
+
 const writeChangeLog = async (rootPath) => {
```

Comment:
I actually think we no longer need to write the CHANGELOG file. This used to be because `standard-version` did not write the first version of the file, but now I think they do. Can you confirm / remove, please?

Reply:
Yes, there is no need to create the changelog file anymore. It is done automatically when doing the first release. 
As a side note, the package `standard-version` is marked as deprecated since this summer (lack of maintainers i think).
But for the moment it is fine i think. And there is a fork that proposes itself as a successor, [see this issue](https://github.com/conventional-changelog/standard-version/issues/919)